components/mintmaker/production/base/manager_patches.yaml

```diff
@@ -17,4 +17,4 @@ spec:
             memory: 256Mi
         env:
         - name: RENOVATE_IMAGE
-          value: "quay.io/konflux-ci/mintmaker-renovate-image:333b7ba"
+          value: "quay.io/konflux-ci/mintmaker-renovate-image:e8ede3a"
```

Comment:
This image has not been related yet, and we need to test it in stage first. The release is still in-progress.